opm/output/eclipse/EclipseIO.cpp

```diff
@@ -192,7 +192,7 @@ inline std::string uppercase( std::string x ) {
 class EclipseIO::Impl {
     public:
     Impl( const EclipseState&, EclipseGrid, const Schedule&, const SummaryConfig& );
-        void writeINITFile( const data::Solution& simProps, const NNC& nnc) const;
+        void writeINITFile( const data::Solution& simProps, std::map<std::string, std::vector<int> > int_data, const NNC& nnc) const;
```

Comment:
The new argument is taken by value instead of const reference. Is that intentional?

Reply:
No - the `Impl` function shoul have been a `const &` - the public function is by intention with value semantics - to support async. That is as with all the IO functions. I suggest we just merge now - this can be fixed with any downstream consequences later.